src/app/modules/map/components/map/map.component.ts

```diff
@@ -378,27 +394,13 @@ export class MapComponent implements OnInit, OnDestroy, AfterViewInit {
           bounds: <any>settings.map.tiles.supportTilesBounds
         }
 
-        const layer = this.createSupportMapTileLayer(supportMaps.name, supportMaps.url, options);
+        const layer = new RegObsTileLayer(supportMaps.url, options);
         layer.setOpacity(supportMaps.opacity);
         layer.addTo(this.layerGroup);
       }
     });
   }
 
-  private createSupportMapTileLayer(name: string, url: string, options: L.TileLayerOptions): RegObsTileLayer {
```

Comment:
Hvorfor har du fjernet denne funksjonen?

Reply:
> Jeg syns det er litt rotete å ha StatensKartverk og NorwegianMixedMap i tillegg til offlinelaget. Det virker unødvendig at det skal lastes norgeskart oppå offline norgeskart, det liker jeg ikke helt. Jeg syns vi må gjøre noe med det, eller ikke gå videre med denne løsningen.

Er helt enig i at det er unødvendig å laste fliser fra Kartverket når vi har dem på telefonen fra før. Tenkte ikke på akkurat denne problemstillingen. Jeg likte bedre den løsningen vi har i test hvor vi kun laster offline-flis hvis vi har denne. Men for å beholde den fine auto-scale-funksjonen må vi vel ha offline-kartet i et separat lag. Lurer på om det er mulig å sjekke (i "online-laget") om vi har offline-flis og la være å laste flisa fra nettet hvis vi har offline-flis tilgjengelig?